src/pages/product.js

```diff
@@ -47,13 +47,9 @@ class ProductPage extends Component {
 
   addToBag () {
     const { variantId, quantity } = this.state
-    this.props.dispatch(addToCart(variantId, parseInt(quantity)))
-      .then(success => {
-        if (success) {
-          this.props.dispatch(toggleMiniBag())
-            .then(setTimeout(() => this.props.dispatch(toggleMiniBag()), 4000))
-        }
-      })
+    this.props.dispatch(addToCart(variantId, parseInt(quantity))).then(success => {
+      if (success) this.props.dispatch(toggleMiniBag(true))
```

Comment:
We are no longer hiding the minibag after 4 seconds because doing it with a `setTimeout` leads to weird behaviour sometimes (e.g. when you close the minibag right after it opens and then open it again it will close itself when `setTimeout` kicks in).